### DISCUSSION TITLE
Error when opening webmail

### DISCUSSION THREAD

**Discussion by nickshanks347** [modoboa/modoboa#2207] [Help]
Hi, just started getting this error when opening webmail either through the modoboa homepage or via <domain name>/webmail.

`Error: Connection to IMAP server failed: [SSL: WRONG_VERSION_NUMBER] wrong version number (_ssl.c:1123)`

Has anyone had this, and if so, how can this be fixed?

**Comment by ogiste** [comment]
Facing this issue as well, were you able to resolve it ?
